thread/src/components/LoginForm.js

```diff
@@ -1,57 +1,75 @@
-                            <Label>Email</Label>
-                            <Input
-                                onChangeText={this.props.loginEmailChanged}
-                                value={this.props.email}
-                            />
-                        </Item>
-                        <Item stackedLabel last>
-                            <Label>Password</Label>
-                            <Input
-                                secureTextEntry
-                                onChangeText={this.props.loginPasswordChanged}
-                                value={this.props.password}
-                            />
-                        </Item>
+
+            <CommonContainer>
+              <Content  style={{ paddingLeft: 20, paddingRight: 20 }}>
+                <Logo />
+                  <Form>
+                      <CommonField
+                          onChangeText={this.props.loginEmailChanged}
+                          value={this.props.email}
+                          style={{marginTop: 30,  marginLeft: -5 }}
+                          label="Username"
+                          />
+                        <CommonField
+                          onChangeText={this.props.loginPasswordChanged}
+                          value={this.props.password}
+                          style={{marginTop: 30, marginLeft: -5}}
+                          secureTextEntry = {true}
+                          label="Password"
+                        />
                     </Form>
                     <LoadingButton
-                        loading={this.props.loading}
-                        style={{ marginTop: 25 }}
-                        onPress={this.onSubmitPress}
-                        text="Login"
+                      loading={this.props.loading}
+                      style={styles.buttonStyle}
+                      onPress={this.onSubmitPress.bind(this)}
+                      text="Login"
+                    />
+                    <LoadingButton
+                      loading={this.props.loading}
+                      style={styles.registerButton}
+                      onPress={() => Actions.register() }
+                      text="Register"
                     />
-                    <Button bordered full style={{ marginTop: 15 }} onPress={() => Actions.register() }>
-                        <Text>Register</Text>
-                    </Button>
-                    <Button transparent full small style={{ marginTop: 15 }} onPress={() => Actions.resetPassword() }>
-                        <Text>Forgot password?</Text>
-                    </Button>
+
+                    <TransparentButton onPress={() => Actions.resetPassword() } label="Forgot Password? Click here!" />
+
                 </Content>
-            </Container>
+            </CommonContainer>
         );
     }
 }
 
+let styles = StyleSheet.create({
+  buttonStyle: {
+    marginTop: 35,
+  },
+  registerButton: {
+      marginTop: 35,
```

Comment:
will need margin bottom, tested with 15px and seemed good with me, however choose what u think is best